components/x-eventpromo/readme.md

```diff
@@ -0,0 +1,16 @@
+# x-eventpromo
+
+This module provides templates for use with [n-eventpromo](https://github.com/Financial-Times/n-eventpromo). 
+Eventpromo is used to promote content from [live.ft.com](http://live.ft.com).
+
+## Installation
+
+This module is compatible with Node 6+ and is distributed on npm.
+
+```bash
+npm install --save @financial-times/x-eventpromo @financial-times/x-engine
+```
+
+This module also requires [`x-engine`][engine] as a peer dependency. The Engine module is used to inject your chosen runtime into the component. Please read the `x-engine` documentation first if you are consuming `x-` components for the first time in your application.
+
+[engine]: https://github.com/Financial-Times/x-dash/tree/master/packages/x-engine
```

Comment:
Usage guidelines and a list of accepted properties needs to be added, see [the readme blueprint](https://github.com/Financial-Times/x-dash/blob/master/private/blueprints/component/readme.md) or check x-teaser for reference.